src/dotnet/Fable.Compiler/Transforms/FSharp2Fable.fs

```diff
@@ -669,6 +671,31 @@ let private transformExpr (com: IFableCompiler) (ctx: Context) fsExpr =
             return sprintf "Cannot compile ILFieldGet(%A, %s)" ownerTyp fieldName
             |> addErrorAndReturnNull com ctx.InlinePath (makeRangeFrom fsExpr)
 
+    // read a member field (Example: Vector3.X)
+    | BasicPatterns.ILFieldGet(Some callee, calleeType, fieldName)
+       when (Replacements.isILMemberFieldSupported (makeType com ctx.GenericArgs calleeType) fieldName) ->
+        let! callee = transformExprOpt com ctx (Some callee)
```

Comment:
You can just use `let! callee = transformExpr com ctx callee` (same below for ILFieldSet).